app/views/api/v1/events/create.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.id @event.id
```

Comment:
Ah, the dreaded new_line missing.

In your Sublime settings, look to set:

    // Set to true to removing trailing white space on save
    "trim_trailing_white_space_on_save": true,

    // Set to true to ensure the last line of the file ends in a newline
    // character when saving
    "ensure_newline_at_eof_on_save": true,

It'll prevent GitHub from complaining about these whitespace things.